scripts/dap-ami-bootstrap.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -e
+
+amazon-linux-extras install -y docker
+systemctl enable docker
+systemctl start docker
+
+if ! docker images | grep appliance; then
+  echo "Loading DAP appliance image into Docker"
+  docker load -i /opt/dap-appliance.tar.gz
+fi
+
+image_id=$(docker images -q)
+container_name='dap'
+
+docker rm -f ${container_name} > /dev/null 2>&1 || true
+
+mkdir -p /var/log/conjur
+mkdir -p /opt/conjur/backup
+
+echo "Creating DAP container"
+cid=$(docker create \
+--name ${container_name} \
+--privileged --restart unless-stopped \
+--log-driver=journald \
+--volume /var/log/conjur:/var/log/conjur:Z \
+--volume /opt/conjur/backup:/opt/conjur/backup:Z \
+--volume /opt/cyberark/dap/security:/opt/cyberark/dap/security:Z \
```

Comment:
It a bit confusion to see `conjur`/`cyberark` but I totally agree with you - it require coordination beyond this project.